AnnoyancesFilter/sections/tweaks.txt

```diff
@@ -259,6 +259,8 @@ newsweek.com##iframe[src="http://sns.europe.newsweek.com/newsletter?popup=1"]
 !######### JS tweaks ##########
 !##############################
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/93895
+filese.me#%#//scriptlet("adjust-setInterval", "countdown", "", "0.02")
```

Comment:
Does this rule work fine on your end?
Btw, can you provide another link? I've had `Bu video telif ihlali sebebiyle kaldırılmıştır.`.